Documentation/running-clair.md

```diff
@@ -77,7 +77,7 @@ If this error is raised, manually execute `docker-compose start clair`.
 $ mkdir $PWD/clair_config
 $ curl -L https://raw.githubusercontent.com/coreos/clair/master/config.yaml.sample -o $PWD/clair_config/config.yaml
 $ docker run -d -e POSTGRES_PASSWORD="" -p 5432:5432 postgres:9.6
```

Comment:
don't we need to add net=host for postgres too?

Reply:
You are right

Reply:
Nope, postgres container publishes 5432 port to host. It is shown to host as localhost at the side of clair container which is running in host network.

Reply:
Ok. I'll approve this change if you can fix the spelling mistake in the commit message.

`Usage of Rrunning clair via docker is wrongly written.` => `Usage of running clair via docker is wrongly written.`

Reply:
I fixed my spelling mistake and changed author to me.